packages/react/src/Card/Card.tsx

```diff
@@ -106,30 +107,32 @@ const CardRoot = forwardRef<HTMLAnchorElement, CardProps>(
     })
 
     const hasIcon = React.Children.toArray(children).some(
-      child => React.isValidElement(child) && typeof child.type !== 'string' && child.type === CardIcon,
+      child => React.isValidElement(child) && typeof child.type !== 'string' && child.type === CardIcon
     )
 
     return (
-      <a
-        href={href}
-        className={clsx(styles.Card, hasIcon && styles['Card--has-icon'], className)}
-        onMouseEnter={handleMouseEnter}
-        onMouseLeave={handleMouseLeave}
-        onFocus={handleOnFocus}
-        onBlur={handleOnBlur}
-        ref={cardRef}
-        {...props}
-      >
-        {filteredChildren}
+      <div className={clsx(styles.Card, hasIcon && styles['Card--has-icon'], className)} ref={cardRef} {...props}>
+        {React.Children.map(filteredChildren, child => {
+          if (React.isValidElement(child) && typeof child.type !== 'string' && child.type === CardHeading) {
+            return React.cloneElement<CardHeadingProps>(child as React.ReactElement<CardHeadingProps>, {
+              onMouseEnter: handleMouseEnter,
+              onMouseLeave: handleMouseLeave,
+              onFocus: handleOnFocus,
+              onBlur: handleOnBlur,
+              href
+            })
+          }
+          return child
+        })}
         <div className={styles.Card__action}>
           <Text as="span" size="300" className={clsx(stylesLink['Link--label'])}>
             {ctaText}
           </Text>
-          <ExpandableArrow className={stylesLink['Link-arrow']} expanded={isHovered || isFocused} />
+          <ExpandableArrow className={stylesLink['Link-arrow']} expanded={isHovered || isFocused} aria-hidden="true" />
```

Comment:
Is it safe to just add `aria-hidden` directly in the ExpandableArrow component? Or are there cases where we want it to be "visible" to ATs?

Reply:
I would say it's okay to directly apply `aria-hidden` to `ExpandableArrow`, as it should never be without context (i.e. `ctaText`), as this would be an issue itself.

I think that `ExpandableArrow` has a `hidden` prop we could use too which should do the same thing.